jscode.js

```diff
@@ -0,0 +1,22 @@
+var words = ['It', 'is', 'awfully', 'hard'];
+var times = [1000, 2000, 4000, 6000];
+
+function showAyah(words,times){
+
+	var htmlElement = document.querySelector(".ayahWords");
+	let wordCount = words.length;
+
+	for(var currentIndex = 0; currentIndex < wordCount; currentIndex++) {
+		
+		(function(index) {

```

Comment:
This function doesn't add any value, remove it. I will show you an example. This is what you wrote:
```js
var n = 0;
function plus(x) {
    n += x;
}
for (var i = 0; i < 10; ++i) {
    plus(x);
}
console.log(n);
```
You don't really need this function. The code above is equivalent to:
```js
var n = 0;
for (var i = 0; i < 10; ++i) {
    n += i;
}
console.log(n);
```